roles/ceph-config/tasks/main.yml

```diff
@@ -135,11 +130,8 @@
     run_once: true
     when:
       - (cephx or generate_fsid)
-      - (not mon_containerized_default_ceph_conf_with_kv and
-          (inventory_hostname in groups.get(mon_group_name, []))) or
-        (not mon_containerized_default_ceph_conf_with_kv and
-          ((groups.get(nfs_group_name, []) | length > 0)
-            and (inventory_hostname == groups.get(nfs_group_name, [])[0])))
+      - ((inventory_hostname in groups.get(mon_group_name, [])) or
+        (groups.get(nfs_group_name, []) | length > 0))
```

Comment:
what about the `and (inventory_hostname == groups.get(nfs_group_name, [])[0]`?

Reply:
@leseb should we simply remove it?